### ISSUE TITLE
Docker Java lambda invocation is broken because of Alpine and JavaFX

### ISSUE THREAD

**Issue by keynmol** [localstack/localstack#534]
I couldn't find the dockerfile for localstack base infrastructure, but by poking around it seems to me that [recent PR](https://github.com/localstack/localstack/pull/518) broke the localstack in docker mode.

If you run a java lambda for DynamoDB events from within docker container, you get this:

```
$ java -cp /opt/code/localstack/localstack/infra/localstack-utils-fat.jar:/tmp/lambda.jar cloud.localstack.LambdaExecutor my.EventHandler /tmp/events_payload.json
11:17:27.237 [main] DEBUG com.amazonaws.AmazonWebServiceClient - Internal logging successfully configured to commons logger: true
Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/javafx/fxml/expression/Expression
	at cloud.localstack.lambda.DDBEventParser.parse(DDBEventParser.java:23)
	at cloud.localstack.LambdaExecutor.main(LambdaExecutor.java:102)
Caused by: java.lang.ClassNotFoundException: com.sun.javafx.fxml.expression.Expression
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 2 more
```

If you run the same command locally (MacOS, for example) using jars copied from the docker container, it actually executes fine. 

I *think* that's because [JavaFX is explicitly removed](localstack/java-maven-node-python) from alpine base jdk package?

**Comment by teroxik** [comment]
Ok, I have checked it out. Adding commits over Christmas was not the best Idea. The static imports provided by Idea were wrong, and the build was successful. We need to improve the Java test coverage.

**Comment by whummer** [comment]
Thanks for reporting @keynmol . This should be fixed in #535 (thanks @teroxik !), new version `0.1.10` of the `localstack-utils` Java library has been released to Maven Central. Closing this ticket now, please comment here if the problem persists. Thanks
